README.md

```diff
@@ -115,23 +118,23 @@ Which one of the following concepts does this best illustrate?
 What is an IIFE, and why might you use it?
 
 **Your answer:**
-
+Immediately invoked function expression - you would use it to call a function immediately after the compiler parses the function
 > ...
 
 ## Question 7
 
 What is the `ui-sref` directive, and how is it used?
 
 **Your answer:**
-
+creates a link in the browser to a view in a SPA
 > ...
 
 ## Question 8
 
 One of the lines of code in the following snippet will throw an error. Which one is it, and why?
 
 **Your answer:**
-
+lines 4 through 7 because they have no braces
 > ...
```

Comment:
The correct answer is line 3. Because we have declared `"use strict";` at the top of the doc, we cannot use global variables, which includes the `i` that is declared in the for loop.